plugins/services/src/js/columns/QuotaOverviewLimitColumn.tsx

```diff
@@ -89,11 +89,70 @@ export function limitRenderer(group: ServiceGroup) {
       <div className="service-table-heading flex-box flex-box-align-vertical-center table-cell-flex-box text-overflow">
         <span className="icon-margin-right">{icon}</span>
         <Trans
-          id={limit}
+          id={limitInfo.limitText}
           render="span"
           className="table-cell-value table-cell-flex-box"
         />
       </div>
     </TextCell>
   );
 }
+
+function itemIsServiceGroup(item: any): item is ServiceGroup {
+  return item.quota && item.quota.limitStatus;
+}
+
+export function getLimitInfoForPod(item: Pod): LimitInfo {
```

Comment:
is this exported intentionally? it looks like `getLimitInfoForService` is good enough as public API.